src/errors.rs

```diff
@@ -0,0 +1,139 @@
+use std::{
```

Comment:
as per our discussion, include a link to sled error with commit hash; to cite them

Reply:
Added.